tests/ecall_ocall/host/host.cpp

```diff
@@ -250,7 +250,7 @@ static void test_cross_enclave_calls()
         enc_cross_enclave_call(enclave_wrap::get(0), &total, 0, SEED, 0);
     OE_TEST(OE_OK == result);
 
-    printf("total=%u, expected_total=%lu\n", total, expected_total);
+    printf("total=%u, expected_total=%zu\n", total, expected_total);
```

Comment:
You know I've been programming with C for a long time and I've never known about `z` in a format specifier. Very cool! I need to use this more often

Reply:
Thanks John! I think it's specifically designed for size_t :)